webhooks/virtualmachine/v1alpha2/validation/virtualmachine_validator_unit_test.go

```diff
@@ -579,68 +591,10 @@ func unitTestsValidateCreate() {
 						}
 					},
 					validate: doValidateWithMsg(
-						`spec.network.routes[0].to: Invalid value: "10.100.10.1": invalid CIDR address: 10.100.10.1`,
-						`spec.network.routes[0].via: Invalid value: "192.168.1": must be an IPv4 or IPv6 address`,
-						`spec.network.routes[1].via: Invalid value: "2463:foobar": must be an IPv4 or IPv6 address`,
-						`spec.network.routes[2]: Invalid value: "": cannot mix IP address families`,
-					),
-				},
-			),
-
-			Entry("validate interfaces",
-				testParams{
-					setup: func(ctx *unitValidatingWebhookContext) {
-						ctx.vm.Spec.Network = vmopv1.VirtualMachineNetworkSpec{
-							HostName: "my-vm",
-							Interfaces: []vmopv1.VirtualMachineNetworkInterfaceSpec{
-								{
-									Addresses: []string{
-										"192.168.1.100/24",
-										"2605:a601:a0ba:720:2ce6:776d:8be4:2496/48",
-									},
-									Gateway4: "192.168.1.1",
-									Gateway6: "2605:a601:a0ba:720:2ce6::1",
-									MTU:      pointer.Int64(9000),
-									Nameservers: []string{
-										"8.8.8.8",
-										"2001:4860:4860::8888",
-									},
-									Routes: []vmopv1.VirtualMachineNetworkRouteSpec{
-										{
-											To:     "10.100.10.1/24",
-											Via:    "10.10.1.1",
-											Metric: 42,
-										},
-										{
-											To:  "fbd6:93e7:bc11:18b2:514f:2b1d:637a:f695/48",
-											Via: "ef71:6ce2:3b91:8349:b2b2:f76c:86ae:915b",
-										},
-									},
-									SearchDomains: []string{"dev.local"},
-								},
-							},
-						}
-					},
-					expectAllowed: true,
-				},
-			),
-
-			Entry("disallow interfaces with default interface",
-				testParams{
-					setup: func(ctx *unitValidatingWebhookContext) {
-						ctx.vm.Spec.Network = vmopv1.VirtualMachineNetworkSpec{
-							HostName:  "my-vm",
-							Addresses: []string{"192.168.1.10/24"},
-							Gateway4:  "192.168.1.1",
-							Interfaces: []vmopv1.VirtualMachineNetworkInterfaceSpec{
-								{
-									DHCP4: true,
-								},
-							},
-						}
-					},
-					validate: doValidateWithMsg(
-						`spec.network.interfaces: Invalid value: "null": interfaces are mutually exclusive with deviceName,network,addresses,dhcp4,dhcp6,gateway4,gateway6,mtu,nameservers,routes,searchDomains fields`,
```

Comment:
Just to confirm, we are removing these since this is now the default behavior already tested above. Feel free to resolve the comment if thats the case.

Reply:
Yes. The fields & assertions tested here don't exist anymore.